doc/environment.yml

```diff
@@ -3,3 +3,4 @@ channels:
   - defaults
 dependencies:
   - rust
+  - python =3.7
```

Comment:
does the single equals actually work!? cool.

Reply:
One for conda, two for PyPI. Because... why be consistent? =P